ethereum/swap_creator_test.go

```diff
@@ -279,6 +279,14 @@ func testClaim(t *testing.T, asset ethcommon.Address, newLogIndex int, value *bi
 		Nonce:        nonce,
 	}
 
+	// ensure we can't claim before setting contract to Ready
+	var s [32]byte
+	secret := proof.Secret()
+	copy(s[:], secret[:])
+	txOpts = *authOrig
+	_, err = contract.Claim(&txOpts, swap, s)
```

Comment:
`proof.Secret()` returns an array and `contract.Claim` accepts an array. The copy on line 285 isn't needed.